lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -34,6 +34,10 @@ def self.cli_to_symbol(cli)
         :aes
       when '7', 7
         :cisco_type_7
+      when 'clear'
+        :clear
```

Comment:
The problem is that :cleartext gets converted to 0, and XR doesn't understand 0.  Looks like :encrypted corresponds to MD5.  How about I remove :clear and :encrypted and just map 0 and 5 to those values in XR (with any other value throwing a NotSupportedError on XR).